libp2p/stream_muxer/mplex/mplex.py

```diff
@@ -34,17 +33,13 @@ class Mplex(IMuxedConn):
     next_channel_id: int
     streams: Dict[StreamID, MplexStream]
     streams_lock: asyncio.Lock
+    new_stream_queue: "asyncio.Queue[IMuxedStream]"
     shutdown: asyncio.Event
 
     _tasks: List["asyncio.Future[Any]"]
 
     # TODO: `generic_protocol_handler` should be refactored out of mplex conn.
```

Comment:
is this comment still relevant?

Reply:
Ah I removed this line in #305 , but I can also fix it here.

Reply:
if you removed it elsewhere, we can leave it here -- just want to keep `master` as clean as possible :)